### PR TITLE
For #8972 - Purple colour matching 

### PR DESCRIPTION
**Pull request by BranescuMihai**
### Pull Request checklist

- [x] **Tests**: This PR only modifies colours
- [x] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [x] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### After merge
- [ ] **Milestone**: Make sure issues finished by this pull request are added to the [milestone](https://github.com/mozilla-mobile/fenix/milestones) of the version currently in development.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

<table>
  <tr>
    <th>History</th>
    <th>Tabs collections</th>
  </tr>
  <tr>
    <td></td>
    <td></td>
  </tr>
</table>

### CODE REVIEW

File: app/src/main/res/color/tab_checkbox_tint.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (ValentinTimisica) [inline app/src/main/res/color/tab_checkbox_tint.xml]: nit: Please add the license header here.

**Comment by ekager** [comment]
Sorry for making you do this, but could you post screenshots in light, dark, and private theme and get Amy to verify all 3 of those before merging?

**Comment by BranescuMihai** [comment]
@ekager sure, I'll check with her before merging! 

<table style="width:100%">
  <tr>
    <th>Light tabs collections</th>
    <th>Light history</th>
  </tr>
  <tr>
    <td></td>
    <td></td>
  </tr>
</table>

<table style="width:100%">
  <tr>
    <th>Night tabs collections</th>
    <th>Night history</th>
  </tr>
  <tr>
    <td></td>
    <td></td>
  </tr>
</table>

**Comment by AmyYLee** [comment]
@BranescuMihai This looks good! Thanks

**Pull request merged**